fe/free_energy.py

```diff
@@ -115,9 +115,10 @@ def prepare_host_edge(self, ff_params, host_system, host_coords):
 # this class is serializable.
 class RelativeFreeEnergy(BaseFreeEnergy):
 
-    def __init__(self, single_topology: topology.SingleTopology, label=None):
+    def __init__(self, single_topology: topology.SingleTopology, label=None, complex_path=None):
         self.top = single_topology
         self.label = label
+        self.complex_path = complex_path
```

Comment:
Nice! Question: is it better for this to be a path, or a different kind of object? (Perhaps making any other choice here would require refactoring `builders.build_protein_system` to accept something other than a path though...)

Reply:
I decided on a path to avoid the pickle files being overly large and to avoid any potential issues with changing how we build protein systems. Will play with this.

Reply:
Makes sense!